recipes/ffbidx/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set version = "1.0.1" %}
+
+{% set cuda_major = (cuda_compiler_version|default("12.0")).rpartition('.')[0] %}
+
+package:
+    name: ffbidx
+    version: {{ version }}
+
+source:
+    url: https://github.com/paulscherrerinstitute/fast-feedback-indexer/archive/refs/tags/v{{ version }}.tar.gz
+    sha256: 99fdd8be09862ceba38436ffb1bf34c164d796e6f88f185b3185d4a606020172
+
+build:
+    number: 0
+    skip: True  # [not linux64]
+    skip: True  # [(cuda_compiler_version or "None") == "None"]
+    skip: true  # [cuda_major != "12"]
+
+requirements:
+    build:
+        - {{ compiler('cuda') }}
+        - {{ compiler('cxx') }}
+        - {{ stdlib('c') }}
+        - cuda-cudart-dev {{ cuda_compiler_version }}
```

Comment:
This should be in host

Reply:
Done

Reply:
Are you sure about this? Don't compilers normally go in build? (because they don't need to be tied to the target architecture, and otherwise would prevent e.g. cross compilation).

Or did you just mean cuda-cudart-dev? (which I agree looks like it should be in host)

Reply:
I am pretty sure that this was a mobile-editing error, and that it was only intended to say that cuda-cudart-dev should be in host. So the compilers and `stdlib("c")` should be in build, not host.